src/packages/@library/ui/carousel/Carousel.svelte

```diff
@@ -1,12 +1,12 @@
 <script lang="ts">
   import { Ellipse } from "@library/icons";
-  export let data = [];
+  export let data: any[] = [];
```

Comment:
How does giving `any` as a data type help? Can we use this opportunity to give better data types?